src/components/hcaptcha/_hcaptcha.scss

```diff
@@ -0,0 +1,59 @@
+/* ===============HCaptcha================== */
+.cmp-adaptiveform-hcaptcha{
+  color: $dark-gray;
+  font-size: $font-md;
+  margin: $field-margin;
+  display: flex;
+  flex-direction: column;
+}
+.cmp-adaptiveform-hcaptcha__widget{
+
```

Comment:
Please check if this is left intentionally blank

Reply:
Yes, we do not need styling for the widget as the rendition is handled by Captcha.

This also adheres to what we have in recaptcha styling, https://github.com/adobe/aem-forms-theme-canvas/blob/main/src/components/recaptcha/_recaptcha.scss